Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM islandoracollabgroup/isle-tomcat:1.1.2
+FROM borndigital/isle-tomcat:1.2.0-dev
```

Comment:
Need to change this to islandoracollabgroup/isle-tomcat:1.2.0

Reply:
Okay I believe that is now fixed. Apologies for missing that in first round.